<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectorySettingsDialog</class>
 <widget class="QDialog" name="DirectorySettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="line1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="done_button">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>110</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
